.circleci/config.yml

```diff
@@ -16,17 +16,27 @@ jobs:
       - run:
           command: |
             source activate ${ENV_NAME}
-            pytest --cov=dask_ml tests
+            mkdir test-reports
+            pytest --cov=dask_ml --junitxml=test-reports/junit.xml tests
```

Comment:
This hooks into circle-ci's output summary. Failing tests will be extracted and put at the top of the output.